engine/src/main/java/org/destinationsol/game/Hero.java

```diff
@@ -19,10 +19,7 @@
 import org.destinationsol.GameOptions;
 import org.destinationsol.assets.audio.OggMusicManager;
 import org.destinationsol.common.SolException;
-import org.destinationsol.game.console.commands.ChangeShipCommandHandler;
-import org.destinationsol.game.console.commands.DieCommandHandler;
-import org.destinationsol.game.console.commands.PositionCommandHandler;
-import org.destinationsol.game.console.commands.RespawnCommandHandler;
+import org.destinationsol.game.console.commands.*;
```

Comment:
Do not use star imports in our projects (see e.g. https://stackoverflow.com/questions/3587071/disable-intellij-starred-package-imports for guide how)